### DISCUSSION TITLE
Linux 6.2 Kernel for Proxmox VE 7.x

### DISCUSSION THREAD

**Discussion by tteck** [tteck/Proxmox#1237] [Announcements]
Opt-in Linux 6.2 Kernel for Proxmox VE 7.x available
https://forum.proxmox.com/threads/opt-in-linux-6-2-kernel-for-proxmox-ve-7-x-available.124189/

**Comment by Coder84619** [comment]
Just as a data point, I switched to the 6.2.6 kernel on a i5-1235U (12th Gen) CPU and all is well. Power usage of the host didn't go down any (6.2 kernels have better Alder Lake power management).
